lua/octo/commands.lua

```diff
@@ -695,4 +695,31 @@ function M.issue_interactive_action(action, kind)
   M.issue_action(action, kind, value)
 end
 
+function M.command_complete(args)
```

Comment:
not sure if we should move it to `completion.lua` for tidiness. Im ok either way

Reply:
Yeah I thought about doing that first, but then `commands` need to be exported. And having that contained in 1 file is kinda nice.

But I'm also fine either way.

Reply:
ok, lets leave it as is. thanks!